tools/kagami/Cargo.toml

```diff
@@ -18,8 +22,13 @@ dev-telemetry = ["iroha_core/dev-telemetry"]
 
 [dependencies]
 iroha_core = { version = "=2.0.0-pre-rc.3", path = "../../core" }
+iroha_crypto = { version = "=2.0.0-pre-rc.3", path = "../../crypto" }
 iroha_config = { version = "=2.0.0-pre-rc.3", path = "../../config" }
+iroha_data_model = { version = "=2.0.0-pre-rc.3", path = "../../data_model" }
+iroha_schema_bin = { version = "=2.0.0-pre-rc.3", path = "../../schema/bin" }
```

Comment:
It has to be in a separate crate so as not to introduce a cyclic dependency. So I called it `iroha_schema_gen` because that's what it is to avoid name conflicts.